setup.py

```diff
@@ -44,10 +46,11 @@ def get_all_extras() -> Dict:
         "click",
         "pyyaml>=4.2b1",
         "jsonschema>=3.0.0",
+        "packaging>=20.3",
         *crypto_deps,
     ]
 
-    cli_gui = ["flask", "connexion[swagger-ui]>=2.4.0", "docker", *cli_deps]
```

Comment:
Are we sure docker is not needed @marcofavorito ?

Reply:
(I guess it was a requirement for oef node)

Reply:
yes not needed for now.

Reply:
(In the future, we may want to support the Ganache Docker image in the test tools.)